packages/auditd/manifest.yml

```
@@ -2,7 +2,7 @@ name: auditd
 title: Auditd
 version: 1.0.0
 release: ga
-description: Auditd Integration
+description: This Elastic integration collects and parses logs from the Audit daemon (`auditd`)
```

We will not be able to interpolate the backticks into code snippets.

Suggestion is to remove. 
```suggestion
description: This Elastic integration collects and parses logs from the Audit daemon (auditd)
```

---

Perfect. I was wondering about that.